src/main/java/com/ericsson/ei/jsonmerge/MergePrepare.java

```diff
@@ -441,6 +443,11 @@ public String addMissingLevels(String originObject, String objectToMerge, String
                 }
             }
         } catch (Exception e) {
+        	log.error("addMissingLevels failed for arguments:");
+        	log.error("originObject was : " + originObject);
+        	log.error("objectTomerge was: " + objectToMerge);
+        	log.error("mergeRule was: " + mergeRule);
+        	log.error("mergePath was: " + mergePath);
```

Comment:
Same thing here as before, i think this should be formatted into the same log.error as we won't be sure of the order otherwise.

Reply:
I agree. Same comment as my last comment regarding this.